src/solc_0.7/Base/TheSandbox712.sol

```diff
@@ -0,0 +1,22 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.7.1;
+
+import {ProxyImplementation} from "../common/BaseWithStorage/ProxyImplementation.sol";
+
+contract TheSandbox712 is ProxyImplementation {
```

Comment:
This should use @atkinsonholly version, that remove ProxyImplementation and make DOMAIN_SEPARATOR public, see https://github.com/thesandboxgame/sandbox-private-contracts/blob/hardhat/src/solc_0.7/Base/TheSandbox712.sol

This is now merged in hardhat